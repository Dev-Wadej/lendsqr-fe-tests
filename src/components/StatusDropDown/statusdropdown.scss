.status-dropdown {
    background: #FFFFFF;
    border: 1px solid rgba(84, 95, 125, 0.04);
    box-shadow: 3px 5px 20px rgba(0, 0, 0, 0.04);
    border-radius: .4rem;
    width: 11.3rem;
    font-size: .9rem;
    font-weight: 500;
    color: #545F7D;
    padding: 1rem 1rem 1rem 1.2rem;
    position: absolute;
    right: -3rem;
    top: 3rem;
    div {
        display: flex;
        align-items: center;
        a {
            display: flex;
            align-items: center;
        }
        span {
            display: inline-block;
            margin-right: .5rem;
        }
    }
    div:nth-child(2) {
        margin: 1rem 0;
    }
}