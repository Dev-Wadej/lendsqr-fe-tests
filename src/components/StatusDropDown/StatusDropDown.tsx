import React from 'react'
import {ReactComponent as Blacklist} from '../../data/svgs/blacklistuser.svg'
import {ReactComponent as ActivateUser} from '../../data/svgs/activateuser.svg'
import {ReactComponent as Eyes} from '../../data/svgs/eyes.svg'
import './statusdropdown.scss'

const StatusDropDown = () => {
  return (
    <section className='status-dropdown'>
      <div>
          <span><Eyes/></span>
          <span>View Details</span>
      </div>
      <div>
          <span><Blacklist/></span>
          <span>Blacklist User</span>
      </div>
      <div>
          <span><ActivateUser/></span>
          <span>Activate User</span>
      </div>
    </section>
  )
}

export default StatusDropDown
