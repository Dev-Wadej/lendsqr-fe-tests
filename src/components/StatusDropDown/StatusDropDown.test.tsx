import StatusDropDown from './StatusDropDown'
import renderer from 'react-test-renderer';
import {cleanup, fireEvent, render} from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';



afterEach(cleanup)
test('Form component snapshot HTML', () => {
  render(
    <MemoryRouter initialEntries={["/users/2"]}>
      <StatusDropDown id='2'/>
    </MemoryRouter>
  );
//   expect(output).toMatchSnapshot();
});