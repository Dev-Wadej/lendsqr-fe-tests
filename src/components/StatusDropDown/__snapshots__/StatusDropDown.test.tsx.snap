// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component snapshot HTML 1`] = `
<section
  className="statuses-dropdown"
>
  <div>
    <a
      href="/2"
      onClick={[Function]}
    >
      <span>
        <svg>
          eyes.svg
        </svg>
      </span>
      <span>
        View Details
      </span>
    </a>
  </div>
  <div>
    <span>
      <svg>
        blacklistuser.svg
      </svg>
    </span>
    <span>
      Blacklist User
    </span>
  </div>
  <div>
    <span>
      <svg>
        activateuser.svg
      </svg>
    </span>
    <span>
      Activate User
    </span>
  </div>
</section>
`;
